libs/designsystem/src/lib/scss/base/_functions.scss

```diff
@@ -157,3 +157,19 @@
     @content;
   }
 }
+
+// Apply click area of at least 44 x 44 px
+// and ignore rounded corners to ensure it is square
+@mixin increase-target-size() {
```

Comment:
I'm not a big fan of mixins without arguments, among other things because their content gets duplicated each time the mixin is used, but that discussion is probably out of scope for this issue. We should discuss the use of mixins at some point, though 😊

What you *could* consider is add arguments for the target size like this:

```scss
@mixin increase-target-size($width, $height: $width) {
...
}
```

It will require that you call the mixin with at least one argument:

```scss
@include increase-target-size($fat-finger-size);
```